src/main/resources/application.yaml

```diff
@@ -35,6 +35,14 @@ spring:
     cache-names: token
     caffeine:
       spec: maximumSize=50,expireAfterWrite=1m
+  cloud:
+    openfeign:
+      httpclient:
+        enabled: false
+      okhttp:
+        enabled: true
```

Comment:
U still need `idam.apachehttpclient.enable` to be set to false.

Reply:
Thanks @adusumillipraveen addressed the above